ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -424,8 +424,16 @@ Redundancy:
 snmp_server:
   contact: < contact_name >
   location: < location >
-  ipv4_access_list: < ipv4-access-list >
-  ipv6_access_list: < ipv6-access-list >
+  ipv4_ACLs: 
+    - name: < ipv4-access-list >
+      vrf: < vrf >
```

Comment:
Does this cover the default (global table) which doesn't have any vrf setting?
Can we reuse the same ACL in global and i.e MGMT?

Reply:
Yes, no VRF implies default VRF, as the command in EOS. Also "default" can be used as VRF value.
Yes, same ACL can be reused in different VRFs but it needs to be defined twice:
  ipv4_ACLs: 
    - name: MyACL
    - name: MyACL
       vrf: MGMT